pkg/sass-parser/lib/src/stringifier.ts

```diff
@@ -165,18 +166,22 @@ export class Stringifier extends PostCssStringifier {
     this.sassAtRule(node, semicolon);
   }
 
+  private ['while-rule'](node: WhileRule): void {
+    this.sassAtRule(node);
+  }
+
   /** Helper method for non-generic Sass at-rules. */
   private sassAtRule(node: postcss.AtRule, semicolon?: boolean): void {
     const start =
       '@' +
       node.name +
       (node.raws.afterName ?? ' ') +
-      node.params +
-      (node.raws.between ?? '');
+      node.params;
```

Comment:
I'm confused as to why `this.block()` doesn't need `between` to be produced first. Does it already do that

Reply:
oh, it does!